app/index-dev.html

```diff
@@ -8,6 +8,7 @@
     <script>
       require("babel-polyfill")
       require("vibrant/dist/Vibrant.min.js")
+      const bravePort = require("./env-echo.js").getEnv("BRAVE_PORT");
```

Comment:
Yeah, I was originally getting a little fancier with it, looking for a port instead of just using the env in which case I didn't want to repeat it just never backed it out all the way here and I was tired... will adjust.